tests/end_to_end/helpers/db.py

```diff
@@ -79,7 +79,7 @@ def run_query_bigquery(query, project):
     client = bigquery.Client(project=project)
     query_job = client.query(query)
     query_job.result()
-    return [r.values() for r in query_job]
+    return [r.values() for r in query_job.result()]
```

Comment:
are you sure it won't return None or empty? because `query_job.result()` is called once before!

Reply:
Removed the previous `query_job.result()` call.
`query_job.result()` returns an iterator from google's core library. And pylint is happy with that.